cloud/kubernetes/README.md

```diff
@@ -115,15 +105,23 @@ Once your kubernetes cluster is up and running, you can launch your cockroach cl
 
 ### Insecure mode
 
-Run: `kubectl create -f cockroachdb-statefulset.yaml`
+To create the cluster, run:
+```shell
+kubectl create -f cockroachdb-statefulset.yaml
+```
+
+Then, to initialize the cluster, run:
+```shell
+kubectl run cockroachdb -it --image=cockroachdb/cockroach --rm --restart=Never -- init --insecure --host=cockroachdb-0.cockroachdb
```

Comment:
I think it would be nice to provide an `cluster_init.yaml` so the secure and insecure instructions are the same except the certificate stuff.

Reply:
https://github.com/cockroachdb/cockroach/pull/20011